tests/suites/test_suite_gcm.function

```diff
@@ -196,6 +267,110 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void gcm_decrypt_and_verify_empty_cipher( int cipher_id,
+                                          data_t * key_str,
+                                          data_t * iv_str,
+                                          data_t * add_str,
+                                          data_t * tag_str,
+                                          int cipher_update_calls,
+                                          int init_result )
+{
+    mbedtls_gcm_context ctx;
+
+    mbedtls_gcm_init( &ctx );
+
+    TEST_ASSERT( mbedtls_gcm_setkey( &ctx, cipher_id, key_str->x, key_str->len * 8 ) == init_result );
```

Comment:
This function doesn't to exercise the case where setkey fails, so `init_result` is not useful. Also goes for the other 3 functions.